pkg/appmanager/as3Manager.go

```diff
@@ -1334,19 +1337,6 @@ func createServiceDecl(cfg *ResourceConfig, sharedApp as3Application) {
 
 	svc.Class = "Service_HTTP"
 
-	for _, prof := range cfg.Virtual.Profiles {
-		switch prof.Name {
-		case "http":
-			svc.ProfileHTTP = as3ResourcePointer{
```

Comment:
I think its not a good idea to remove the http profile from virtual server. Can we look for an alternative?